.github/workflows/check.yml

```diff
@@ -28,7 +28,8 @@ jobs:
     runs-on: ubuntu-latest
     env:
       BEERUS_TEST_RUN: 1
-      BEERUS_TEST_STARKNET_URL: https://starknet-mainnet.g.alchemy.com/starknet/version/rpc/v0_7/${{ secrets.ALCHEMY_KEY }}
```

Comment:
Test context is currently network-agnostic, why not just keep `BEERUS_TEST_STARKNET_URL` and set it to the network-specific URL for specific test run?

Reply:
Oh, I see - the subset of tests is expected to run against testnet.